docs/release_notes/5.0.28-24.md

```diff
@@ -0,0 +1,53 @@
+# Percona Server for MongoDB 5.0.28-24 ({{date.5_0_28}})
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 5.0.28-24 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.x Community Edition. 
+
+Percona Server for MongoDB 5.0.28-24 includes improvements and bug fixes of [MongoDB 5.0.28 Community Edition](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.28---jul-15--2024) and supports its protocols and drivers.
+
+## Release Highlights
+
+This release of Percona Server for MongoDB includes the following features and improvements:
+
+### Enhanced Telemetry for better product usage reporting
+
+The enhanced telemetry feature provides comprehensive information about how it works, its components and metrics as well as updated methods how to disable telemetry. Read more in [Telemetry on Percona Server for MongoDB](../telemetry.md)
+
+### Reduce mean time to resolve (MTTR) compromised encryption key incidents in KMIP
+
+Starting with this release, Percona Server for MongoDB automatically activates the master encryption keys at startup and periodically checks (polls) their status in a KMIP server. If a master encryption key for a node is not in the Active state, the node reports an error and shuts down. This method allows security engineers to quickly identify which nodes require out-of-schedule master key rotation, such as in the case of compromised keys, without needing to rotate keys for the entire cluster. 
+
+The following diagram provides the sample master key lifecycle with key state polling:
+
+![image](../_images/kmip-state-polling.jpg)
```

Comment:
@radoslawszulgo Isn't it too much for release notes? IMHO, it'll better work on respective feature page if user wants to learn more about this new feature.

Reply:
Too much? Maybe. Let's see. If this attracts at least 1 person it's good. I'll try to lookup if there were some clicks to learn more about KMIP from the Release Notes page. 

The problem is that the image can't be opened in wider screen or with ability to zoom in. It's currently hard to read.